### ISSUE TITLE
Quick Navigation To Util Tests Does Not Work

### ISSUE THREAD

**Issue by nadnoslen** [Turbo87/intellij-emberjs#122]
It doesn't appear that this plugin respects _utils_ in the same way that it respects other elements of EmberJS.

My `app/utils/` classes are hard to find using `CMD-O` _class search_.  In addition to that, I cannot quick-navigate (CMD-SHIFT-T) back and forth between a util class and its test.  I'm sure there are other symptoms of utility classes not working as expected; those are the two I bump into the most often.

**Comment by dwickern** [comment]
You're right, we don't index the modules in `/app/utils`. Utils should be included in the search since it's an official module type: https://ember-cli.com/user-guide/#module-directory-naming-structure

We will need define `UTIL` in [EmberFileType](https://github.com/Turbo87/intellij-emberjs/blob/1e2eb7247d0ed4a12bcc40d259d5ee3877e8b8ab/src/main/kotlin/com/emberjs/EmberFileType.kt) and I think increment the version in [EmberNameIndex](https://github.com/Turbo87/intellij-emberjs/blob/1e2eb7247d0ed4a12bcc40d259d5ee3877e8b8ab/src/main/kotlin/com/emberjs/index/EmberNameIndex.kt#L20)

**Comment by Turbo87** [comment]
@dwickern the "issue" is that utils are usually not used through the container, so something like `util:foo-bar` doesn't make much sense. it appears that my initial thought of modeling these things after the container references wasn't such a great idea after all...

**Comment by nadnoslen** [comment]
They're not much different from _mixins_ yet you appear to be indexing those?

**Comment by SBlanc42** [comment]
I'd like to take a shoot at this one, as it seems pretty easy and I'm a beginner, however, I dont understand what the version incrementation is for. Could anyone help me understand why it is needed ?

**Comment by Turbo87** [comment]
@SBlanc42 I'm not sure what you mean by "version incrementation" :thinking:

**Comment by SBlanc42** [comment]
@dwickern suggested that it would be needed to increment EmberNameIndex

**Comment by dwickern** [comment]
You increment the version to force IntelliJ to rebuild the index
